cmd/prometheus/main.go

```diff
@@ -256,8 +262,10 @@ func (c *flagConfig) setFeatureListOptions(logger *slog.Logger) error {
 				parser.ExperimentalDurationExpr = true
 				logger.Info("Experimental duration expression parsing enabled.")
 			case "native-histograms":
+				features.Disable(features.FeatureFlags, o)
 				logger.Warn("This option for --enable-feature is a no-op. To scrape native histograms, set the scrape_native_histograms scrape config setting to true.", "option", o)
 			case "ooo-native-histograms":
+				features.Disable(features.FeatureFlags, o)
```

Comment:
Why disable? 
```suggestion
```

Reply:
This is in the features_flag section, indicating an obsolete flag.